cloud_governance/main/test_env.yaml

```diff
@@ -0,0 +1,167 @@
+policy: ""
```

Comment:
I think it should be env.yaml

Reply:
The one should copy to env.yaml and pass only defined values.